LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2019 Hannes Winkler
+Copyright (c) 2021 Hannes Winkler
```

Comment:
the FSF says you shouldn't change the year, but include all the years changes were made to the files. i.e.
```suggestion
Copyright (c) 2019-2021 Hannes Winkler
```

but either way it's not that important AFAICT